### PR TITLE
DOCS-2929: Update changelog

### PR DESCRIPTION
**Pull request by sguequierre**
* updates changelog for mid july-sept 6th

### CODE REVIEW

**Review by JessamyT — APPROVED** [review]
LGTM % delete the docs changes entry, and two other small suggestions

File: docs/appendix/changelog.md [hunk 1]
```diff
@@ -19,24 +19,58 @@ outputs:
...
+You can now create, upload and submit custom training scripts to the Viam Registry.
+For more information, see [Create custom training scripts](/how-tos/create-custom-training-scripts/).
+
+{{% /changelog %}}
 
 {{% changelog date="2024-07-19" color="changed" title="Operators can now view data" %}}
 
 The operator role now has view permissions for the data in the respective resource a user has access to.
-For more information, see [/cloud/rbac/#data-and-machine-learning].
+For more information, see [Data and machine learning](/cloud/rbac/#data-and-machine-learning).
```
Comment (npentrel) [inline docs/appendix/changelog.md:67]: That link text may be confusing without added specificity.
```suggestion
For more information, see [Data and machine learning permissions](/cloud/rbac/#data-and-machine-learning).
```

File: docs/appendix/changelog.md [hunk 2]
```diff
@@ -19,24 +19,58 @@ outputs:
...
 
-The vision service now supports two new methods: [`CaptureAllFromCamera`](/services/vision/#captureallfromcamera) and [`GetProperties`](/services/vision/#getproperties).
+`viam-micro-server` now supports cameras on ESP32s.
+For more information, see [Configure an esp32-camera](/components/camera/esp32-camera/).
+
+{{% /changelog %}}
+
+{{% changelog date="2024-08-26" color="changed" title="Micro-RDK now called viam-micro-server" %}}
+
+The micro-RDK is now referred to as `viam-micro-server`.
```
Comment (npentrel) [inline docs/appendix/changelog.md:38]:
```suggestion
The lightweight version of `viam-server` that is built from the micro-RDK is now referred to as `viam-micro-server`.
```

File: docs/appendix/changelog.md [hunk 3]
```diff
@@ -19,24 +19,58 @@ outputs:
...
+
+Data capture is now possible for the vision service.
+For more information, see [Supported components and services](/services/data/capture/#supported-components-and-services).
 
-{{% /changelog %}}-->
+{{% /changelog %}}
+
+{{% changelog date="2024-08-01" color="added" title="Create custom training scripts" %}}
+
+You can now create, upload and submit custom training scripts to the Viam Registry.
```
Comment (npentrel) [inline docs/appendix/changelog.md:59]: That doesn't make sense, you submit jobs not trainign scripts
```suggestion
You can now upload custom training scripts to the Viam Registry and use them to train machine learning models.
```

**Review by npentrel — COMMENTED** [review]
some additional minor changes.

File: docs/appendix/changelog.md (hunk 2 above)
Comment (sguequierre) [reply docs/appendix/changelog.md:31]: good suggestion!

File: docs/appendix/changelog.md (hunk 3 above)
Comment (sguequierre) [reply docs/appendix/changelog.md:52]: thanks

**Review by npentrel — APPROVED** [review]

**Pull request merged**